docs/DeveloperGuide.md

```diff
@@ -721,3 +721,21 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+## **Appendix: Planned Enhancements**
+**Team size**: 5
+
+1. **Make find command shorter by allowing fuzzy search of module-role, such that user can find all contacts with roles related to a specified module, just by inputting module code or even fraction of it.
+(For example:`find r/CS2103T` finds all contacts related to CS2103T, `find r/CS` finds all contacts related to modules whose module code starts with CS).**
+The current find command automatically assumes that user wants to search student role if only a module code is supplied to the find command, which may be counter-intuitive to some users.
+Sometimes, user cannot remember the full module code, or want to search over a wider range of modules, but the current implementation does not support that either,
+hence reducing the usefulness of this command.
+
+![point1 screenshot](images/Planned_Enhancements_Screenshots/Point1_1.png)
+(As shown in the screenshot, the result of `find r/CS1101S` only matches contacts with role **CS1101S Student**, despite that there are 2 professors of CS1101S in the sample data.)
+
+![point1 screenshot](images/Planned_Enhancements_Screenshots/Point1_2.png)
+(As showin the screenshot, the result of `find r/CS` results in error messages because construction of search query require exact module code to work.)
```

Comment:
```suggestion
(As shown the screenshot, `find r/CS` results in error messages because construction of search query require a full module code to work.)
```

Reply:
I think can leave `exact` because it is true that CS1231S only matches for CS1231S and CS1231 only matches CS1231, in this case CS1231 is a full module code but it is not exactly the mode matching the CS1231S people